lib/utils/analytics/generatePayload.js

```diff
@@ -51,6 +52,13 @@ module.exports = async (serverless) => {
     );
   })();
 
+  let timezone;
+  try {
+    timezone = Intl.DateTimeFormat().resolvedOptions().timeZone; // eslint-disable-line new-cap
```

Comment:
I think we can put `new` and remove `// eslint-disable-line new-cap` comment.

I assume this, as some other JS interfaces can be initialized both with and without `new`

Reply:
correct :+1: